docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -69,11 +69,19 @@ List of node IDs with which this node will not peer. The node ID is the public k
 bootnodes=["enode://c35c3...d615f@1.2.3.4:30303","enode://f42c13...fc456@1.2.3.5:30303"]
 ```
   
+```bash tab="Example Bootnode Command Line"
```

Comment:
Ambiguous title given the options name, better named "Example command line when acting as a bootnode"